system-reinstall-bootc/src/users.rs

```diff
@@ -183,4 +257,22 @@ mod test {
         assert!(result.contains("root"));
         assert!(result.contains("foo-doe"));
     }
+
+    fn split_on_first_whitespace(input: &str) -> Option<(&str, &str)> {
+        let mut parts = input.splitn(2, char::is_whitespace);
+        let first = parts.next()?;
+        let second = parts.next().unwrap_or("");
+        Some((first, second))
+    }
+
+    #[test]
+    pub(crate) fn test_me() {
+        println!("ahhh");
+        let input = "foo   bar  bar2 bar3";
+        if let Some((first, rest)) = split_on_first_whitespace(input) {
+            println!("First part: '{}'", first);
```

Comment:
For stuff like this I use `dbg!` - it's actually quicker to type *and* we deny shipping it by default in `make validate-rust` so that one can't accidentally commit debug prints (I recently did this in bootupd :see_no_evil: https://github.com/coreos/bootupd/pull/845 )